spec/features/collection_drafts/forms/invalid_picklists_spec.rb

```diff
@@ -416,34 +417,6 @@
     end
   end
 
-  context 'when viewing the Platform Type field' do
```

Comment:
Since you made the functionality for invalid picklist values to work for platforms, do should we keep the tests for it, or is it not necessary?